src/gamma_inc.jl

```diff
@@ -611,3 +906,96 @@ gamma_inc(a::Float16,x::Float16,ind::Integer) = ( Float16(gamma_inc(Float64(a),F
 gamma_inc(a::Real,x::Real,ind::Integer) = (gamma_inc(float(a),float(x),ind))
 gamma_inc(a::Integer,x::Integer,ind::Integer) = gamma_inc(Float64(a),Float64(x),ind)
 gamma_inc(a::AbstractFloat,x::AbstractFloat,ind::Integer) = throw(MethodError(gamma_inc,(a,x,ind,"")))
+
+#EFFICIENT AND ACCURATE ALGORITHMS FOR THECOMPUTATION AND INVERSION OF THE INCOMPLETEGAMMA FUNCTION RATIOS by Amparo Gil, Javier Segura, Nico M. Temme
+#SIAM Journal on Scientific Computing 34(6) (2012), A2965-A2981
+# arXiv:1306.1754
+
+"""
+    gamma_inc_inv(a,p,q)
+    
+DLMF: https://dlmf.nist.gov/8.2#E4 , https://dlmf.nist.gov/8.2#E5
+Wiki: https://en.wikipedia.org/wiki/Incomplete_gamma_function
+
+gamma_inc(a,x) or (P(a,x),Q(a,x)) is the Incomplete gamma function ratio given by : ``1/\\Gamma (a) \\int_{0}^{x} e^{-t}t^{a-1} dt``
+gamma_inc_inv(a,p,q) inverts the gamma_inc function, by computing `x` given `a`,`p`,`q` in P(a,x)=p and Q(a,x)=q.
+"""
+function gamma_inc_inv(a::Float64, p::Float64, q::Float64)
+    if p < 0.5
+        pcase = true
+        porq = p
+        s = -1
+    else
+        pcase = false
+        porq = q
+        s = 1
+    end
+    m = 0
```

Comment:
lets make `m` a Bool everywhere, and maybe give it a better name, e.g. `haseta`?
```suggestion
    haseta = false
```